doc/data-files-description.md

```diff
@@ -9,89 +9,90 @@ This document contains information about all data files associated with this pro
+|`pbta-snv-mutect2.vep.maf.gz` | PBTA data file | Somatic mutation calling [Workflow](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_strelka2_mutect2_manta_workflow.cwl) | Somatic SNV - Mutect2 [annotated MAF file](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/doc/format/vep-maf.md) |
+|`pbta-snv-strelka2.vep.maf.gz` | PBTA data file | Somatic mutation calling [Workflow](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_strelka2_mutect2_manta_workflow.cwl) | Somatic SNV - Strelka2 [annotated MAF file](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/doc/format/vep-maf.md) |
+|`pbta-snv-vardict.vep.maf.gz` | PBTA data file | Somatic mutation calling [Workflow](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc-vardict-wf.cwl) | Somatic SNV - VarDict [annotated MAF file](https://github.com/AlexsLemonade/OpenPBTA-analysis/blob/master/doc/format/vep-maf.md) |
+|`tcga-snv-consensus-snv.maf.tsv.gz` | Analysis file | [`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers) | TCGA Consensus calls for SNVs and small indels made from Mutect2, Strelka2, and Lancet. |
+|`tcga-snv-mutation-tmb-all.tsv` | Analysis file | [`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers) | Tumor Mutation burden calculations using all mutations identified by both of Mutect2 and Strelka2 throughout the genome. |
+|`tcga-snv-mutation-tmb-coding.tsv` | Analysis file | [`snv-callers`](https://github.com/AlexsLemonade/OpenPBTA-analysis/tree/master/analyses/snv-callers) | Tumor Mutation burden calculations using coding only mutations identified by both Mutect2 and Strelka2 only within coding sequence regions of the genome. |
+|`pbta-star-log-final.tar.gz` | PBTA data file | Gene expression abundance estimation [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl) | STAR log final output files |
+|`pbta-star-log-manifest.tsv` | PBTA data file | Gene expression abundance estimation [Workflow](https://github.com/kids-first/kf-rnaseq-workflow/blob/052e31844711fa5cd0ef5e581618e9d7bba3fec6/workflow/kfdrc_RNAseq_workflow.cwl) | File to map STAR output to biospecimen IDs |
+|`pbta-sv-manta.tsv.gz` | PBTA data file | Structural variant calling [Workflow](https://github.com/d3b-center/OpenPBTA-workflows/blob/master/cwl/kfdrc_strelka2_mutect2_manta_workflow.cwl) | Somatic Structural Variant - Manta output, annotated with AnnotSV (WGS samples only) |
+|`pbta-tcga-manifest.tsv` | PBTA data file | Somatic mutation calling | Manifest of TCGA tumor/normal BAMs used for SNV calling, Tumor_Sample_Barcodes, and histologies; information retrieved from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/) |
```

Comment:
Let's re-arrange this a bit given that there is no originating script, and the current info in that column makes it look like a workflow link is missing - 

```suggestion
|`pbta-tcga-manifest.tsv` | PBTA data file | Retrieved from [GDC website API endpoint](https://docs.gdc.cancer.gov/API/Users_Guide/Search_and_Retrieval/) | Manifest of TCGA tumor/normal BAMs used for SNV calling, Tumor_Sample_Barcodes, and histologies |
```